src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -874,8 +874,11 @@ void Lowering::LowerModPow2(GenTree* node)
         BlockRange().InsertAfter(cnsZero, cmp);
         LowerNode(cmp);
 
-        mod->ChangeOper(GT_CNEG_LT);
-        mod->gtOp1 = trueExpr;
+        mod->SetOper(GT_SELECT_NEGCC);
```

Comment:
I think this should keep using `ChangeOper`.

Reply:
Done